src/onboarding/containers/LoginPage.tsx

```diff
@@ -56,7 +57,10 @@ export const LoginPage: FC = () => {
   } else {
     if (isFlagEnabled('universalLogin')) {
       if (CLOUD) {
-        const url = new URL(`${CLOUD_QUARTZ_URL}`).href
+        let url
+        getUiproxyQuartzLoginUrl({}).then(response => {
+          url = response.data
```

Comment:
the redirect will also need to go inside the `then`'s callback function. the `then` isn't like an `await` where the code after it gets run after it resolves, this code will run in this order:

1. `getUiproxyQuartzLoginUrl` makes a request
1. `console.warn` is fired
1. `window.location.replace(url)` is called
1. `url gets set to response.data`

Reply:
Ahhh I see! Okay i'll fix that!